src/gui/dialogs/statistics_dialog.cpp

```diff
@@ -154,34 +156,35 @@ void statistics_dialog::add_damage_row(
 
 	const long long shifted = ((expected * 20) + shift) / (2 * shift);
 	std::ostringstream str;
-	str << damage << " / "
-		<< static_cast<double>(shifted) * 0.1
-		<< spacer // TODO: should probably make this two columns
-		<< (((dsa < 0) ^ (expected < 0)) ? "" : "+")
-		<< (expected == 0 ? 0 : 100 * dsa / expected) << '%';
-
+	str << damage << " / " << static_cast<double>(shifted) * 0.1;
 	item["label"] = str.str();
 	data.emplace("damage_overall", item);
 
 	str.str("");
+	str << (((dsa < 0) ^ (expected < 0)) ? "" : "+")
+		<< (expected == 0 ? 0 : 100 * dsa / expected) << '%';
+	item["label"] = str.str() + spacer;
```

Comment:
Just make the spacer a part of the string stream so that you don't need to append it afterwards.